tests/01_microtest.R

```diff
@@ -1,18 +1,29 @@
 # detach(package:MoBiToolboxForR)
+# install.packages("C:/workspace/sw/MoBiToolBoxForR_7.1.0.zip", repos = NULL, type = "win.binary", lib = "C:/workspace/sw/TBforRVERSIONS/7.1.0/")
```

Comment:
I agree. It can not be avoided to have an absolute path to the library, "C:/workspace/sw/TBforRVERSIONS/7.1.0.7", as this is independent of the installation of the tests. Others (below "R-Toolbox") can be changed to relative paths.